DeviceTests/DeviceTests.Shared/Battery_Tests.cs

```diff
@@ -1,46 +1,36 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using Xamarin.Essentials;
+﻿using Xamarin.Essentials;
 using Xunit;
 
 namespace DeviceTests
 {
+    // TEST NOTES:
+    //   - the battery is not emulated on iOS simulators
```

Comment:
I want to make sure we add proper notes to the test so that we all know why the test was skipped.